web/satellite/src/components/projectsList/ProjectsList.vue

```diff
@@ -0,0 +1,180 @@
+// Copyright (C) 2020 Storj Labs, Inc.
```

Comment:
nit - 2021. This comment goes for all other new files and their copyright headers